corehq/apps/builds/jadjar.py

```diff
@@ -132,7 +134,7 @@ def pack(self, files, jad_properties=None):
         buffer = StringIO(self.jar)
         with ZipFile(buffer, 'a', ZIP_DEFLATED) as zipper:
             for path in files:
-                zipper.writestr(path, files[path])
+                zipper.writestr(path, files[path].replace(BASE_ADDRESS, J2ME_ADDRESS))
```

Comment:
Kind of hacky, but it is nice that we don't have duplicate files running around.